content/docs/user-guide/components/reference/gradients/fastnoise-gradient.md

```diff
@@ -24,15 +44,23 @@ Add the **FastNoise Gradient** component to use one of the noise generation algo
 | Property | Description | Values | Default |
 |-|-|-|-|
 | **Generate Random Seed** | Sets the **Random Seed** property below to a random value. | | |
-| **Preview** | Displays the output gradient of this component after all properties are applied. | | |
+| **Preview** | Displays the output gradient of this component after all properties are applied. Click the previewer icon in the top-left of the preview image to show a larger preview of the gradient in a dockable window. | | |
 | **Pin Preview to Shape** | Sets an entity with a compatible Shape component to use as the bounds of the preview if **Constrain to Shape** is `Enabled`. | EntityId | Current Entity |
 | **Preview Position** | Sets the world location of the preview.<br> <br>*This field is available only if there is no entity selected in **Pin Preview to Shape**.* | Vector3: -Infinity to Infinity | X:`0.0`, Y:`0.0`, Z:`0.0` |
 | **Preview Size** | Sets the dimensions of the preview. | Vector3: 0.0 to Infinity | X:`1.0`, Y:`1.0`, Z:`1.0` |
 | **Constrain to Shape** | If `Enabled`, the gradient preview uses the bounds of the entity selected in **Pin Preview to Shape**.<br> <br>*This field is available only if an entity is selected in **Pin Preview to Shape**.* | Boolean | `Disabled` |
 | **Noise Type** | Sets the noise generation algorithm used to generate the gradient. | `Value`, `Value Fractal`, `Perlin`, `Perlin Fractal`, `Simplex`, `Simplex Fractal`, `Cellular`, `White Noise`, `Cubic`, or `Cubic Fractal` | `Perlin Fractal` |
-| **Random Seed** | Sets the seed value for the noise generation algorithm. | Integer: 1 to Infinity | `1` |
-| **Frequency** | Sets the frequency of the generated noise. Larger values result in noise that is more coarse.  | Float: 0.0001 - Infinity | `1.0` |
-| **FastNoise Advanced Settings - Interpolation** | Sets the function used to smooth between gradient values. | `Linear`, `Hermite`, or `Quintic` | `Quintic` |
+| **Random Seed** | Sets the initialization value for the pseudorandom noise generation algorithm. Each value generates a different pattern of noise. | Integer: 1 to Infinity | `1` |
+| **Frequency** | Sets the frequency of the generated noise. Smaller values result in noise that is dilated and larger values result in noise that is condensed.  | Float: 0.0001 - Infinity | `1.0` |
+| **FastNoise Advanced Settings - Interpolation** | Sets the function used to smooth between gradient values. Refer to the following section for [Interpolation type descriptions and examples](#interpolation-type-examples). | `Linear`, `Hermite`, or `Quintic` | `Quintic` |
```

Comment:
The link to `interpolation-type-examples` only works on this first tab now. If you go to the Value Fractal tab, for example, and click on the same link, it scrolls to the top - because it's looking for the section on the first tab and it can't find it because it is hidden. To make this work, you'd have to give each copy of the interpolation type examples its own unique reference link, like `{#value-fractal-interpolation-type-examples}`.

Same problem with the link to fractal type examples on Perlin Fractal and other tabs.